pkg/local_object_storage/writecache/persist.go

```diff
@@ -71,7 +64,7 @@ func (c *cache) persistToCache(objs []objectInfo) []int {
 			if err != nil {
 				return err
 			}
-
+			c.dbSize.Add(uint64(len(objs[i].data)))
```

Comment:
It is not a good idea to do this inside `db.Update`, because if tx fails it will be rollbacked and we will end up with an incorrect counter. So we better do this in a local variable and update `dbSize` after `db.Update`.

Reply:
Done.

Reply:
Isn't this `c.dbSize.Add()` still inside `db.Update`?